lib/interactor/builder.ex

```diff
@@ -0,0 +1,130 @@
+defmodule Interactor.Builder do
+
+  @moduledoc """
+
+
+  The Interactor.Builer module functionality and code is **heavily** influenced
+  and copied from the Plug.Builder code.
+  TODO.
+
+  Example:
+
+      def Example.CreatePost do
+        use Interactor.Interaction
+        import Ecto.Changeset
+
+        interactor :post_changeset
```

Comment:
I was thinking about this DSL the other day and I think `interactor` here is kind of misleading. It seems like it's really a step within an interator, rather than actually calling an interactor itself. Is there a reason you went with that instead of a verb like `perform` or `execute` or a noun like `step` or `action`?